src/appleseedmaya/extensionattributes.cpp

```diff
@@ -248,6 +253,29 @@ namespace
         AttributeUtils::makeInput(numAttrFn);
         modifier.addExtensionAttribute(nodeClass, attr);
 
+        attr = createNumericAttribute<bool>(
+            numAttrFn,
+            "asCastIndirectLight",
+            "asCastIndirectLight",
+            MFnNumericData::kBoolean,
+            false,
+            status);
+        AttributeUtils::makeInput(numAttrFn);
+        modifier.addExtensionAttribute(nodeClass, attr);
+
+        attr = createNumericAttribute<float>(
+                   numAttrFn,
```

Comment:
Fix indentation.

Reply:
This was removed in https://github.com/appleseedhq/appleseed-maya/commit/55b39fd209498104cbe8218fae8d163fcdcef9d4.